### ISSUE TITLE
Share profile between multiple websites

### ISSUE THREAD

**Issue by zebhlindqvist** [linuxmint/webapp-manager#268]
I like the isolate profile feature too keep websites separate.

Another useful thing would be to share a profile between a set of websites.

In my case I would use one profile for all Microsoft websites like Teams and Outlook, and another profile for all Google websites like Gmail and Drive.

This way I'm guessing you would only have to login once for all Microsoft websites and once for all Google websites. I don't want to mix all of them in a single profile to achieve this.

**Comment by TipsyTheCat** [comment]
yeah, it's really frustrating logging in many times like this. it should by default share cookies with the browser and have an "isolate app" option when adding an app.

**Comment by soulstenance** [comment]
I think it'd be better from a privacy standpoint to default to the current behavior (isolation), but give the option to disable this if the user doesn't care. I personally don't want or need my web apps talking to each other.

For Google, I recommend just using Google Containers addon in the regular browser, which will automatically yoink all Google services and sites into their own container, isolating it from the rest of your websites.

**Comment by ironhak** [comment]
Any news?? I hate to re-login every time
